README.md

```diff
@@ -5,32 +5,30 @@
 
 ## Quickstart
 
-Run Data explorer with a test dataset:
+Run local Data explorer with a test dataset:
 
-* `cd api/dataset_config && ln -s test_dataset current`
-* `cd ../.. && docker-compose up --build`
+* `docker-compose up --build`
 * Navigate to `localhost:4400`
 
-To use a different dataset:
-
-* For each dataset, there is a directory in `api/dataset_config`, e.g.
-  `api/dataset_config/amp_pd`.
-    * If you used https://github.com/DataBiosphere/data-explorer-indexers to
-      index your dataset, we recommend moving your dataset config directory from
-      that repo to `api/dataset_config/MY_DATASET`. If you need to run
-      more commands in the `data-explorer-indexers` repo, you can simply refer
-      to `api/dataset_config/MY_DATASET` in this repo.
-* Create symlink: `cd api/dataset_config && ln -s MY_DATASET current`
-* Index your data into an Elasticsearch started by
-  `docker run -p 9200:9200 docker.elastic.co/elasticsearch/elasticsearch-oss:6.2.2`. You can use one of the indexers at
-  https://github.com/DataBiosphere/data-explorer-indexers, or any other indexer.
-  https://github.com/DataBiosphere/data-explorer-indexers explains how to
-  set up the config files. Specifically:
-  * There must be a file named `dataset.json` that has a `name` field. This
-    determines the name of the Elasticsearch index.
-  * There must be a file named `facet_fields.csv` with the `readable_field_name`
-    column filled out.
-* `docker-compose up --build`
+## Run local Data explorer with a specific dataset
+
+This repo contains an API server that reads from Elasticsearch, and a UI server
```

Comment:
This bit is describing how our code works, so would probably belong better in the Architecture section so we can keep this to just a set of instructions. Unless you want to make a comment about how our docker-compose can use an existing Elasticsearch instance as long as you provide the config, and defaults to a test dataset otherwise. That would be fine here.

Reply:
Alternatively, if you wanted this to be a sort of one liner overview, it might be good to put something similar to this at the top of the README:

The Data Explorer is an interface for easily visualizing and examining data from an Elasticsearch index.

Reply:
Rephrased.